doc/releases/changelog-dev.md

```diff
@@ -42,6 +42,11 @@
 * Added the `QutritBasisState` operator to support qutrit state preparation for the `default.qutrit` device
   [(#4185)](https://github.com/PennyLaneAI/pennylane/pull/4185)
 
+* Added dunder methods to `FermiWord` and `FermiSentence` to allow arithmetic operations 
+  using `+`, `-` and `*` as well as in-place addition, subtraction and multiplication, between 
```

Comment:
Do we have in-place addition in this PR? If not, we will add it in future PRs.

Reply:
The in-place versions seem to work based on the `__add__`, `__sub__` and `__mul__` methods, so I haven't added them. I can add tests for them, there are no tests currently. But in my Jupyter notebook on my computer they work. 

Is there some scenario where they need to be explicitly defined that I'm not aware of?

Reply:
They are not necessary for this PR. I suggest removing "as well as in-place addition" to prevent confusion.